### PR TITLE
Initial release of Linux-PAM

### PR DESCRIPTION
**Pull request by GaetanCambier**
for future integration of squid (pam_auth module)

### CODE REVIEW

File: cross/Linux-PAM/PLIST [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+bin:sbin/unix_chkpwd
+bin:sbin/unix_update
+rsc:share/locale/de/LC_MESSAGES/Linux-PAM.mo
+rsc:share/locale/en_GB/LC_MESSAGES/Linux-PAM.mo
+rsc:share/locale/es/LC_MESSAGES/Linux-PAM.mo
+rsc:share/locale/fr/LC_MESSAGES/Linux-PAM.mo
+rsc:share/locale/it/LC_MESSAGES/Linux-PAM.mo
+rsc:share/locale/nl/LC_MESSAGES/Linux-PAM.mo
+rsc:share/locale/pt_BR/LC_MESSAGES/Linux-PAM.mo
+rsc:share/locale/pt/LC_MESSAGES/Linux-PAM.mo
```
Comment (Diaoul) [inline cross/Linux-PAM/PLIST]: Please remove all share stuff and *.la files

**Comment by Diaoul** [comment]
Please keep the name in cross/ lowercase for consistency

**Comment by GaetanCambier** [comment]
done

**Comment by Diaoul** [comment]
Forgot to mention, please remove share/locale stuff too. We won't need that on the NAS.

**Comment by GaetanCambier** [comment]
done

**Pull request merged**